packages/dds/sequence/src/test/revertibles.spec.ts

```diff
@@ -169,14 +283,187 @@ describe("Sequence.Revertibles with Remote Edits", () => {
 		const id = collection.add(0, 7, IntervalType.Simple).getIntervalId()!;
 		containerRuntimeFactory.processAllMessages();
 
-		sharedString2.insertText(5, " hi");
 		collection.removeIntervalById(id);
+		sharedString2.insertText(5, " hi");
 		containerRuntimeFactory.processAllMessages();
 
 		revertIntervalRevertibles(sharedString, revertibles.splice(0));
 		containerRuntimeFactory.processAllMessages();
 
 		assert.equal(sharedString.getText(), sharedString2.getText());
 		assertIntervals(sharedString, collection, [{ start: 0, end: 10 }]);
+		assertIntervals(sharedString2, collection2, [{ start: 0, end: 10 }]);
+	});
+	it("remote interval change interacting with reverting an interval remove", () => {
+		sharedString.insertText(0, "hello world");
+		// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+		const id = collection.add(0, 5, IntervalType.Simple).getIntervalId()!;
+		containerRuntimeFactory.processAllMessages();
+
+		collection.on("deleteInterval", (interval, local, op) => {
+			appendLocalDeleteToRevertibles(sharedString, interval, revertibles);
+		});
+		collection.removeIntervalById(id);
+		containerRuntimeFactory.processAllMessages();
+
+		collection2.change(id, 3, 8);
```

Comment:
I think you want to remove the previous processAllMessages, which causes client2 to see the interval remove so the change is a no-op.

Reply:
Although it's going to be a no-op on client 1 anyway.

Reply:
Maybe the test is more interesting if you swap the order of the remove and change.